README.md

```diff
@@ -478,14 +445,12 @@ results = mq.index("my-first-multimodal-index").search(
         "What are some vehicles and modes of transport?": 1.0,
         "Aeroplanes and other things that fly": -1.0,
     },
-    searchable_attributes=["captioned_image"],
 )
 print("\nQuery 2:")
 pprint.pprint(results)
 
 results = mq.index("my-first-multimodal-index").search(
     q={"Animals of the Perissodactyla order": -1.0},
```

Comment:
delete the comma at the end of the line

Reply:
Thanks Pandu for the comments. Also, deleted the unnecessary commas in other places